ConcernsCaseWork/Service.Redis/NtiWarningLetter/INtiWarningLetterCachedService.cs

```diff
@@ -12,5 +12,7 @@ public interface INtiWarningLetterCachedService
 		Task<ICollection<NtiWarningLetterDto>> GetNtiWarningLettersForCaseAsync(long caseUrn);
 		Task<NtiWarningLetterDto> GetNtiWarningLetterAsync(long ntiWarningLetterId);
```

Comment:
Get and Patch aren't used

Reply:
these will be used in the stories immediately following this story.